src/components/Timeline/TimelineUpdate.tsx

```diff
@@ -3,12 +3,33 @@ import TimelineJourneyInstance from './updates/TimelineJourneyInstance';
 import TimelineJourneyMilestone from './updates/TimelineJourneyMilestone';
 import TimelineJourneyStart from './updates/TimelineJourneyStart';
 import TimelineNoteAdded from './updates/TimelineNoteAdded';
+import TimelineTags from './updates/TimelineTags';
 import { UPDATE_TYPES, ZetkinUpdate } from 'types/updates';
 
+import { ZetkinUpdateTags } from '../../types/updates';
+
 interface Props {
   update: ZetkinUpdate;
 }
 
+// Type predicate function that checks if the action (second part) of update type
+// matches any of the supplied types, and if so returns true indicating that
+// the update is of type T.
+function isWildcardType<T extends ZetkinUpdate>(
```

Comment:
Like 🔥 
I feel like there are other places in the code where using type predicates would make things clearer, especially for new devs. I can't think of any right now, it's just a hunch 😅

Reply:
Yeah aren't they nice!? I love me a type predicate, but rarely get around to using them. 😞